### ISSUE TITLE
ui-auth:3.2.0 gradle sync fails due to lint error

### ISSUE THREAD

**Issue by imax531** [firebase/FirebaseUI-Android#1116]
### Step 1: Are you in the right place?

  * yes

### Step 2: Describe your environment

  * Android device: N/A
  * Android OS version: N/A
  * Google Play Services version: 11.8.0
  * Firebase/Play Services SDK version: 11.8.0
  * FirebaseUI version: 3.2.0
  * Android Studio version: 3.0.1
  
### Step 3: Describe the problem:

When trying to add auth-ui gradle dependency, gradle sync fails with the following error:

```
Failed to resolve: firebaseui-android.internal:lintchecks:unspecified
```

#### Steps to reproduce:

  1. Open a new project with Android Studio
  2. Add to the application build.gradle file the following lines:
   ```
dependencies {
    // ...
    implementation  'com.google.firebase:firebase-auth:11.8.0'
    implementation  'com.google.android.gms:play-services-auth:11.8.0'
    implementation  'com.firebaseui:firebase-ui-auth:3.2.0'
    //...
}
   ```
  3. Sync gradle
  
#### Observed Results:

  The sync failed with the error message
  ```
  Failed to resolve: firebaseui-android.internal:lintchecks:unspecified
  ```

#### Expected Results:

  The sync should have succeeded
  
#### Relevant Code:

build.gradle (Module: app)
```
apply plugin: 'com.android.application'

android {
    compileSdkVersion 27
    defaultConfig {
        applicationId "com.roytmax.myapplication"
        minSdkVersion 21
        targetSdkVersion 27
        versionCode 1
        versionName "1.0"
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
}

dependencies {
    implementation fileTree(dir: 'libs', include: ['*.jar'])
    implementation 'com.android.support:appcompat-v7:27.0.2'
    implementation 'com.android.support.constraint:constraint-layout:1.0.2'

    implementation 'com.google.firebase:firebase-auth:11.8.0'
    implementation 'com.google.android.gms:play-services-auth:11.8.0'
    implementation 'com.firebaseui:firebase-ui-auth:3.2.0'

    testImplementation 'junit:junit:4.12'
    androidTestImplementation 'com.android.support.test:runner:1.0.1'
    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
}
```

builde.gradle (Project: MyApplication)
```
// Top-level build file where you can add configuration options common to all sub-projects/modules.

buildscript {
    
    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.0.1'
        

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}

allprojects {
    repositories {
        google()
        jcenter()
    }
}

task clean(type: Delete) {
    delete rootProject.buildDir
}
```

#### Misc:
* I attempted to use compile instead of implementation but the problem persists.
```
    compile 'com.google.firebase:firebase-auth:11.8.0'
    compile 'com.google.android.gms:play-services-auth:11.8.0'
    compile 'com.firebaseui:firebase-ui-auth:3.2.0'
````
* The sync succeeds with `com.firebaseui:firebase-ui-auth` versions 3.1.2 and 3.1.3.
* I attempted to use the entire library, but the same error occurs
```
compile com.firebaseui:firebase-ui:3.2.0
```
* Syncing other sections of the firebase ui (storage/database/firestore) works just fine with version 3.2.0.
* Related issue: #125

**Comment by SUPERCILEX** [comment]
Shucks! We really need to be more careful about our modules... working on a fix right now. In the meantime, this should work:
```groovy
compile('com.firebaseui:firebase-ui-auth:3.2.0') {
    exclude module: 'lintchecks'
}
```

**Comment by samtstern** [comment]
This has been fixed and released in `3.2.1`.
